run/README.namelist

```diff
@@ -637,6 +641,12 @@ Namelist variables for controlling the adaptive time step option:
  alevsiz                             = 12  ! for Tegen aerosol input levels, set automatically 
  no_src_types                        = 6   ! for Tegen aerosols: organic and black carbon, sea salt, sulfalte, dust,
                                              and stratospheric aerosol (volcanic ashes - currently 0), set automatically
+ use_rap_aero_icbc                   = .true.   ! Option to ingest real-time data containing aerosols (default is false – off; new in 4.4)
+ qna_update                          = 1   ! Update time-varying sfc aerosol emission from climatology or real-time data 
+                                             with mp_physics = 28. Use with input file ‘wrfqnainp_d0*’
+                                             (must set auxinput17_interval and io_form_auxinput17). Default is 0=off (new in 4.4)
+ wif_fire_emit                       = .true.   ! Option to include biomass burning organic and black carbon aerosols with mp_physics = 28 (new in 4.4)
+ wif_fire_inj                        = 1   ! (default) vertically distribute biomass burning emissions in mp_physics = 28 (new in 4.4)
```

Comment:
@kkeene44 It may be better to move these 4 namelists right after use_aero_icbc as it applies to mp_physics = 28 only. Also set the values of use_rap_aero_icbc and wif_fire_emit to false, as they are the defaults in Registry. Can also modify comment for use_aero_icbc to be 'use climatological aerosol input from WPS'.

Reply:
@weiwangncar Okay, I made those changes and committed them.